cmd/kube-apiserver/app/server.go

```diff
@@ -299,7 +299,18 @@ func Run(s *options.ServerRunOptions) error {
 	}
 
 	admissionControlPluginNames := strings.Split(s.GenericServerRunOptions.AdmissionControl, ",")
-	pluginInitializer := kubeadmission.NewPluginInitializer(client, sharedInformers, apiAuthorizer)
+	var cloudConfigReader *os.File
+
+	if s.CloudProvider.CloudConfigFile != "" {
+		cloudConfigReader, err = os.Open(s.CloudProvider.CloudConfigFile)
+		if err != nil {
+			glog.Errorf("Error opening cloud configuration file %s: %#v", s.CloudProvider.CloudConfigFile, err)
```

Comment:
If someone passes the cloud config argument and it's not a file, that should already have been an error.  It is better to treat this as a hard error to prevent future debugging and head scratching of "but i specified the cloud provider, why isn't it being used?".